modal/experimental/__init__.py

```diff
@@ -85,6 +85,19 @@ async def list_deployed_apps(environment_name: str = "", client: Optional[_Clien
     return app_infos
 
 
+@synchronizer.create_blocking
+async def stop_app(name: str, *, environment_name: Optional[str] = None, client: Optional[_Client] = None) -> None:
+    """Stop a deployed App.
+
+    This interface is experimental and may change in the future,
+    although the functionality will continue to be supported.
+    """
+    client_ = client or await _Client.from_env()
+    app = await _App.lookup(name, environment_name=environment_name, client=client_)
```

Comment:
`modal app stop` ends up using this RPC:

https://github.com/modal-labs/modal-client/blob/d99c43f0666de72781ca47dfa49dacea9f652825/modal/cli/utils.py#L52

while this PR and `_App.lookup` ends up calling: 

https://github.com/modal-labs/modal-client/blob/d99c43f0666de72781ca47dfa49dacea9f652825/modal/app.py#L311

Should these be consistent?